File: demo/src/main/java/com/baozi/demo/persenter/DemoPresenter.java
package com.baozi.demo.persenter;

import com.baozi.demo.ui.at.BaseAt;
import com.baozi.mvp.presenter.BasePresenter;

public class DemoPresenter extends BasePresenter<BaseAt> {


    @Override
    public void cancelNetWork() {

    }

    @Override
    public void netWorkError(Throwable throwable) {
        mView.getLoadHelper().showError(throwable);
    }
}

File: module-kotlindemo/src/main/java/com/baozi/module_kotlindemo/presenter/DemoPresenter.kt
package com.baozi.module_kotlindemo.presenter

import com.baozi.module_kotlindemo.model.DemoView
import com.baozi.mvp.presenter.BasePresenter

/**
 * Created by baozi on 2017/10/20.
 */
class DemoPresenter : BasePresenter<DemoView>() {

    override fun onCreate() {
//        mView.context
//        mView.getIntent()
        mView.A()
        mView.B()
    }

    override fun initData() {
    }

    override fun cancelNetWork() {
    }
}